arq/connections.py

```diff
@@ -195,10 +195,12 @@ async def _get_job_def(self, job_id: bytes, score: int) -> JobDef:
         jd.score = score
         return jd
 
-    async def queued_jobs(self, *, queue_name: str = default_queue_name) -> List[JobDef]:
+    async def queued_jobs(self, *, queue_name: Optional[str] = None) -> List[JobDef]:
```

Comment:
You can change the behavior (like you have) without changing this type annotation. 

The change here is wrong.

Reply:
While mypy doesn't warn with `queue_name: str`, `reveal_type(queue_name)` shows `arq/connections.py:202: note: Revealed type is "Union[builtins.str, None]"` and mypy does warn in strict mode. From a quick grep using `: Optional[...] = None` is more common than `: ... = None` so it would not even be matching style.

Reply:
Right now your implementation is based off `if queue_name is None`, but your type annotation says it cannot be `None`.  
If you look at `enqueue_job`'s implementation you see what I mean.

Reply:
Ah shit, I read this the second I woke up and apparently are color blind. My bad.. Your implementation is correct.